tests/integration/Newsletter/ShortcodesTest.php

```diff
@@ -136,76 +136,53 @@ public function testItCanProcessNewsletterShortcodes() {
   public function itCanProcessPostNotificationNewsletterNumberShortcode() {
     // create first post notification
     $postNotificationHistory = $this->_createNewsletter(
-      $parentId = $this->newsletterId,
+      $parentId = $this->newsletter,
```

Comment:
This assignment is from old code probably, but it's unnecessary. You used refactor, maybe, and it's the reason why you didn't notice. Similar problem ins on lines 148-149.